client/src/lib/src/types/requests.ts

```diff
@@ -11,15 +11,17 @@ export enum RequestType {
   getLinksInDocument = 'getLinksInDocument',
   ProcessRecipeScanResults = 'ProcessRecipeScanResults',
   GetVar = 'getVar',
-  GetAllVar = 'getAllVar'
+  GetAllVar = 'getAllVar',
+  GetPossibleSrcUris = 'getPossibleSrcUris'
```

Comment:
A more accurate name would be `getRecipeLocalFiles`. SRC_URI can also contain remote URLs like http://, git://, ...
What we return here are the recipe's local files, accessible by file://